apps/block_scout_web/assets/js/pages/address/fee_payments.js

```diff
@@ -0,0 +1,119 @@
+import $ from 'jquery'
+import omit from 'lodash.omit'
+import humps from 'humps'
+import numeral from 'numeral'
+import socket from '../../socket'
+import { batchChannel } from '../../lib/utils'
+import { connectElements } from '../../lib/redux_helpers.js'
+import { createAsyncLoadStore } from '../../lib/async_listing_load'
+import '../address'
+import { isFiltered } from './utils'
+
+const BATCH_THRESHOLD = 10
+
+export const initialState = {
+  channelDisconnected: false,
+  addressHash: null,
+  filter: null,
+  feePaymentsBatch: []
+}
+
+export function reducer (state, action) {
+  switch (action.type) {
+    case 'PAGE_LOAD':
+    case 'ELEMENTS_LOAD': {
+      return Object.assign({}, state, omit(action, 'type'))
+    }
+    case 'CHANNEL_DISCONNECTED': {
+      if (state.beyondPageOne) return state
+
+      return Object.assign({}, state, {
+        channelDisconnected: true,
+        feePaymentsBatch: []
+      })
+    }
+    case 'RECEIVED_NEW_FEE_PAYMENT_BATCH': {
+      if (state.channelDisconnected || state.beyondPageOne) return state
+
+      const incomingFeePayments = action.msgs
+        .filter(({ toAddressHash, fromAddressHash }) => (
+          !state.filter ||
```

Comment:
Sounds good!

Reply:
If it's not being used, it could be confusing to have. This code could or could not work if pointed to an indexing explorer instance

Reply:
because live-updates is not working in the read-only instances this def won't be used. I see your point about it being confusing but it's really not a bad thing to have because it will force us to ensure it works when we get live-updates working.